src/python/build_management/revisions.py

```diff
@@ -620,9 +620,9 @@ def get_first_revision_in_list(revision_list):
   if not min_revision:
     return first_revision
 
-  for index in range(len(revision_list)):
-    if revision_list[index] >= min_revision:
-      return revision_list[index]
+  for revision_list_item in revision_list:
```

Comment:
Why not just revision, that is simpler than revision_list_item

Reply:
sure, will fix it